safe/definitions/keyword_properties.py

```diff
@@ -0,0 +1,314 @@
+# coding=utf-8
+
+"""Definitions relating to layer keywords."""
+
+from safe.utilities.i18n import tr
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+# Base Metadata Property
+property_organisation = {
+    'key': 'property_organisation',
+    'name': tr('Organisation'),
+    'description': tr('An organized body of people who own the layer.')
+}
+
+property_email = {
+    'key': 'property_email',
+    'name': tr('Organisation'),
+    'description': tr('The email address of the author of the layer.')
+}
+
+property_date = {
+    'key': 'property_date',
+    'name': tr('Date'),
+    'description': tr('The date when the layer is created.')
+}
+
+property_abstract = {
+    'key': 'property_abstract',
+    'name': tr('Abstract'),
+    'description': tr(
+        'A brief narrative summary of the content of the layer.')
+}
+
+property_title = {
+    'key': 'property_title',
+    'name': tr('Title'),
+    'description': tr('A name of the layer.')
+}
+
+property_license = {
+    'key': 'property_license',
+    'name': tr('License'),
+    'description': tr('A permit from an authority to use the layer.')
+}
+
+property_url = {
+    'key': 'property_url',
+    'name': tr('URL'),
+    'description': tr(
+        'The address of World Web Page where we can find the layer or its '
+        'description.')
+}
+
+property_layer_purpose = {
+    'key': 'property_layer_purpose',
+    'name': tr('Layer Purpose'),
+    'description': tr(
+        'The purpose of the layer, it can be hazard layer, exposure layer, or '
```

Comment:
:D we use comma in Bahasa Indonesia, if you have more than 2 objects (like in English)